services/product-service/service.yaml

```diff
@@ -54,7 +54,7 @@ Resources:
                   LogConfiguration:
                     LogDriver: awslogs
                     Options:
-                        awslogs-group: !Ref AWS::StackName
+                        awslogs-group: CloudWatchLogsGroup
```

Comment:
Should this not say "!Ref CloudWatchLogsGroup" vs CloudWatchLogsGroup?

Reply:
Yes, thanks! Added.